expressEngine/login/service/service.js

```diff
@@ -0,0 +1,61 @@
+// import service libraries
+import { BaseService, DBService } from '../../lib/service';
+
+// import collections
+import { User } from '../../model';
+import { userToken } from '../../model';
```

Comment:
You have to import userToken from just above import statement because there is duplicate import from `../../model`

Reply:
done